src/CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+# Contributing
+
+Thank you for your interest in improving the Super Rentals Tutorial! This repository contains the logic to generate the prose markdown for our tutorial, as well as the code for the tutorial _app_ itself.
+
+Please note that all of the content to generate markdown and code are located in individual chapter files in `/src/chapters`. Any modifications/additions should occur there.
+
+When building this tutorial you may notice that you have a `/dist/chapters` generated locally. Please note that this directory is _not_ committed, and contains the output of your locally-generated tutorial content. Please be mindful not to modify anything in the `/dist` directory as it is git ignored and any changes there will not impact what is generate! Instead, make your changes inside of the `/src` directory only.
+
+## Making changes to the tutorial prose
+
+If you are modifying or adding new prose to the tutorial, please be sure to follow the formatting style guide below! Following these conventions will make it easier to review your contribution. 😊
+
+### Formatting style guide
+
+* When naming chapters, be sure that they are prefixed with two digits before the chapter name to ensure that they are ordered correctly.
+    * For example: `01-orientation.md`.
+* When referring to filenames, be sure to specify the relatiive path to file. This is to make it easy to copy-paste the filename when people use the tutorial.
+    * For example: *"Let's start by creating a new file at `app/components/jumbo.hbs`."*
+* When referring to other chapters, link to the chapter using its relative path in the tutorial.
+    * For example: `We learned about this in a [previous chapter](../02-building-pages/)`.
```

Comment:
The trailing slash is important. Otherwise it will fail lint on the guides repo.